### ISSUE TITLE
Merge some addon syntaxes to skript

### ISSUE THREAD

**Issue by Anarchick** [SkriptLang/Skript#3269]
Since many years skript merge the content of an addon to himself if it is useful. I propose a list of addon feature wich should be merge to skript to prevent break of addon.
I know that devellopers spend a lot of time to doing this and they can not agree with this merge, so I recommand to this dev to make a PR himself.

I have try to list only things that are present on the spigot API via methods. (except total experience of player from skbee , but it's very usefull even if it can be write in a script !!) and not NMS or plugin dependency.

Tell me if some things should not be merge and I will edit my text

- [ ] Exp : All blocks within %location% and %location% (cuboid not raytrace) from skQuerry  or skBee (https://github.com/SkriptLang/Skript/issues/3243)
- [ ] Exp: Addon list from skUtilities
- [ ] Exp: [all] [registred] [script] commands from Tuske
- [ ] Exp: [get] (size|number|amount) of dropped %entity% from skellett
- [ ] Exp: Anvil repair cost from skellett
- [ ] All armor change from skBee
- [ ] Exp: Armor value from skRayFall
- [ ] Exp: Arrow critical state and knockback strength from skQuerry
- [ ] Exp: Art from skQuerry
- [ ] World border from skQuerry
- [ ] Bossbar 
- [ ] Breeding from skellett
- [ ] Brewing from skellett
- [ ] Exp: Command info from Tuske
- [ ] Exp: complicated math from skQuerry (as expression and not function)
- [ ] Exp: destination on teleport event from MundoSk (as event-location)
- [ ] Exp: %objects% that are not from MundoSk
- [ ] Ender crystal parameters from skellett
- [ ] Exp: entity far away state from skellett
- [ ] Exp: entity ID from skellett
- [ ] entity block storage from skBee
- [ ] regex
- [ ] fishing expressions from skellett
- [ ] Exp: gender of %player% from Tuske
- [ ] Exp: glowing itemstack from skQuerry
- [ ] Exp: health scale from skQuerry
- [ ] Exp: hidden item flags from skBee
- [ ] Horse color and style from Tuske
- [ ] Exp: ignored sleep state from skellett
- [ ] Exp: insta break from skellett (on block damage)
- [ ] Exp: items of recipe from Tuske
- [ ] Exp: item within entity from skellett
- [ ] Exp: jukebox record from Tuske
- [ ] Exp: last color of %string% from Tuske
- [ ] Exp: llama décor from skellett
- [ ] Exp: loaded chunks from skQuerry
- [ ] Number of loaded variables/commands/events/expressions/conditions/effect/types/functions/scripts from skUtilities
- [ ] Exp: loggin result from mundosk
- [ ] https://papermc.io/javadocs/paper/1.15/org/bukkit/entity/Minecart.html
- [ ] Exp: new uuid from skQuerry
- [ ] Exp: all operators from skellett
- [ ] Exp: owner of skull from MundoSK
- [ ] Exp: plugin list from skutilities
- [ ] Exp: plugin of command from skLib
- [ ] Exp: plugin version from skQuerry
- [ ] Exp: primed tnt fuse time from skellett
- [ ] Exp: primer from skQuerry
- [ ] Exp: rabbit type from Tuske
- [ ] Ram expression
- [ ] Exp: random text from skQuerry
- [ ] Exp: slime size form skellett
- [ ] Exp: spectral arrow glowing time form skellett
- [ ] Exp: stored enchantments from skBee
- [ ] Exp: system time from skUtilities
- [ ] Exp: target reason from skellett
- [ ] Exp: tile entities of %chunk% from skQuerry
- [ ] Exp: total experience from skBee
- [ ] Exp: index %number% of %objects% from skQuerry
- [ ] world %string%

**Comment by ShaneBeee** [comment]
The one thing I dont like about lists this long is you have say 100 different things you want, and slowly over time they get done.
Then for instances its 2 months later, 3-4 things are checked off, and before you know it, this issue is so old it gets forgotten.

**Comment by Anarchick** [comment]
#1291

**Comment by FranKusmiruk** [comment]
I am also against this kind of issues unless they provide substantial details such as recommended implementations or just technical information that would be useful for us the developers or collaborators. There's even some of the features that are already in Skript.

That said, I am closing the issue as it is hard to follow and the 1 feature request per issue basis is preferable.
